src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -208,4 +210,31 @@ public void parseMatric_validMatric_success() throws ParseException {
     public void parseMatric_invalidMatric_throwsParseException() {
         assertThrows(ParseException.class, () -> ParserUtil.parseMatric(INVALID_MATRIC));
     }
+
+    @Test
+    public void parseMatric_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> ParserUtil.parseMatric(null));
+    }
+
+    @Test
+    public void parseStudio_validStudio_success() throws ParseException {
+        seedu.address.model.student.Studio expected = new Studio(VALID_STUDIO);
+        assertEquals(expected, ParserUtil.parseStudio(VALID_STUDIO));
+    }
+
+    @Test
+    public void parseStudio_invalidStudio_throwsParseException() {
+        assertThrows(ParseException.class, () -> ParserUtil.parseStudio(INVALID_STUDIO));
+    }
+
+    @Test
+    public void parseStudio_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> ParserUtil.parseStudio(null));
+    }
+
```

Comment:
Very insightful and comprehensive unit testing. Lets strive to maximize code coverage